keploy.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+installKeploy (){
+    IS_CI=false
+    for arg in "$@"
+    do
+        case $arg in
+            -isCI)
+                IS_CI=true
+                shift
+            ;;
+            *)
+            ;;
+        esac
+    done
+
+    install_keploy_arm() {
+        curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_linux_arm64.tar.gz" | tar xz -C /tmp
+
+        sudo mkdir -p /usr/local/bin && sudo mv /tmp/keploy /usr/local/bin
+        alias keploy='sudo -E env PATH=$PATH keploy'
+    }
+
+    install_keploy_amd() {
+        curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_linux_amd64.tar.gz" | tar xz -C /tmp
+
+        sudo mkdir -p /usr/local/bin && sudo mv /tmp/keploy /usr/local/bin
+
+        alias keploy='sudo -E env PATH=$PATH keploy'
+    }
+
+    install_colima_docker() {
+        if ! docker network ls | grep -q 'keploy-network'; then
+            docker network create keploy-network
+        fi
+        alias keploy='docker run --pull always --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v "$(pwd)":/files -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock --rm ghcr.io/keploy/keploy'
```

Comment:
Here you have to make sure that alias is always updated , It is better to see some approach where you can scrap/get alias from like our kelpoy website or somewhere. I mean this can be implemented later but I guess current alias is not updated , there are some changes done by @PranshuSrivastava please connect with him and update it.